packages/extension-browser/src/content-script/connector.ts

```diff
@@ -10,13 +10,12 @@ import {
     restoreReferences,
     traverse
 } from '@hint/utils-dom';
-
 import {
-    ConnectorOptionsConfig,
     IConnector,
     FetchEnd,
     NetworkData
 } from 'hint/dist/src/lib/types';
+import { ConnectorOptionsConfig } from '@hint/utils/dist/src/types/config';
```

Comment:
`/dist/src/types/...`  😞

Can we make the types a top level of the packages? Is this part of one of the pending `utils` issues we have opened?

Reply:
The are at the top level of the packages, but this is the extension, and the utils package, so we need to use the full path.

Reply:
I believe that in talking with @antross we were OK to link to the root path now that everything was split even if that was a small bump on the extension size.

Can you give it a try and see how much bigger it becomes?

Reply:
I didn't try lately, but the error we use to have using `@hint/utils`  wasn't only the size, it was that the build of the extension fails. But I will try again

Reply:
we can use `@hint/utils` only if we are importing types, but if we are importing a method, then we need to use the path to the file.